webextension/scripts/utils.js

```diff
@@ -460,8 +460,12 @@ function notify(message, callback) {
 }
 
 function checkLastError() {
-  if (chrome.runtime.lastError && chrome.runtime.lastError.message.startsWith("No tab with id:")) {
+  if (chrome.runtime.lastError) {
+    if (chrome.runtime.lastError.message.startsWith("No tab with id:")){
       console.log(chrome.runtime.lastError.message);
```

Comment:
That's fine with me, although I would have thought we want to _ignore_ such errors (i.e. not log them)

Reply:
Yes, we can ignore them. I will remove the console. statement. I added it for testing purpose.